src/components/login/loginForm.js

```diff
@@ -108,9 +107,9 @@ class LoginForm extends React.Component {
   }
 
   devPreFill() {
-    const address = Cookies.get('address');
-    const passphrase = Cookies.get('passphrase');
-    const network = parseInt(Cookies.get('network'), 10) || 0;
+    const address = localStorage.getItem('address') || '';
+    const passphrase = localStorage.getItem('passphrase') || '';
```

Comment:
The `|| ''` it's only for avoid `null` in `<Input />` (because `localStorage.getItem` return `null` if value not exist)